src/feed-list.scss

```diff
@@ -0,0 +1,102 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: 0;
+    margin-right: 0.5rem;
+
+    @include fd-rtl() {
+      margin-left: 0.5rem;
+      margin-right: 0;
+    }
+  }
+
+  &__body + &__body {
+    border-top: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem $fd-feed-list-padding-end 1rem $fd-feed-list-padding-start;
+  }
+
+  &__actions {
+    @include fd-reset();
+
+    height: 2rem;
+  }
+
+  .#{$block}__icon,
+  &__icon-title {
+    color: $fd-feed-list-icon-color;
+  }
+
+  &--group {
+    flex-direction: column;
+  }
+
+  &--link {
+    cursor: pointer;
+  }
+
+  .#{$block}__name {
+    @include fd-reset();
+
+    font-weight: bold;
+  }
+
+  .#{$block}__text {
+    @include fd_reset();
+
+    &:hover {
+      .#{$block}__more {
+        color: var(--sapLinkColor);
+      }
+    }
+  }
+
+  .#{$block}__more {
+    @include fd-reset();
+
+    text-transform: uppercase;
+    color: grey;

```

Comment:
You can't have a hard-coded value for the color. The theming won't work. You need to use a variable.